tv_apps/fling-player/test/unit/casting_message_template.js

```diff
@@ -0,0 +1,78 @@
+(function(exports) {
+  'use strict';
+
+  var castingMsgTemplate = {
+    get : function () {
```

Comment:
As you've made it a function, how about just making the template you need to choose as an argument?